src/components/stepforms/StepFormHeader.vue

```diff
@@ -1,33 +1,41 @@
 <template functional>
   <div class="step-edit-form">
-    <button class="step-edit-form__back-button" @click="parent.cancelEdition">
-      <i class="fas fa-angle-left" aria-hidden="true" />
-      BACK
-    </button>
-    <div class="step-edit-form__title-container">
-      <h1>{{ props.title }}</h1>
-      <a
-        :href="`https://weaverbird.toucantoco.com/docs/${props.stepName}`"
-        target="_blank"
-        rel="noopener"
-        :data-version="props.version"
-      >
-        <i class="fas fa-question-circle" aria-hidden="true" />
-      </a>
+    <div class="step-edit-form__container">
+      <button class="step-edit-form__back-button" @click="parent.cancelEdition">
+        <i class="fas fa-angle-left" aria-hidden="true" />
+        BACK
+      </button>
+      <div class="step-edit-form__title-container">
+        <h1>{{ props.title }}</h1>
+        <a
+          :href="`https://weaverbird.toucantoco.com/docs/${props.stepName}`"
+          target="_blank"
+          rel="noopener"
+          :data-version="props.version"
+        >
+          <i class="fas fa-question-circle" aria-hidden="true" />
+        </a>
+      </div>
+      <div class="step-edit-form__empty" />
+    </div>
+    <div v-if="props.backendError" class="step-edit-form__error">
+      <strong>{{ props.backendError }}</strong>
```

Comment:
Did you have a specific reason to put it in he header of the form rather than outside in a specific component ?
(Both works, I'm just curious :p)

Reply:
Less code, introduce a component directly in stepform need to add it to every steps. It's just more text to add so a bigger package but we could totally do it and it will be easier to change if necessary